Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -98,6 +98,35 @@ public static class NetworkClient
         // scene loading
         public static bool isLoadingScene;
 
+        [RuntimeInitializeOnLoadMethod]
+        static void Init()
```

Comment:
is there a reason why there's an extra Init() instead of reusing the existing ShutDown() which already clears state?

if the [runtimeinit] tag works then that's a great solution to fast enter playmode tho

Reply:
This was merged into Shutdown, which was renamed to ResetStatics per discord convo